includes/class-wc-payments.php

```diff
@@ -594,7 +594,7 @@ public static function init_rest_api() {
 		$settings_controller->register_routes();
 
 		include_once WCPAY_ABSPATH . 'includes/admin/class-wc-rest-payments-reader-controller.php';
-		$charges_controller = new WC_REST_Payments_Reader_Controller( self::$api_client );
+		$charges_controller = new WC_REST_Payments_Reader_Controller( self::$api_client, self::$card_gateway );
```

Comment:
I'd prefer to require and inject the new service here, not a big fan of static methods, which are helpers actually.

Reply:
Done!